sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java

```diff
@@ -323,6 +326,41 @@ public List<SclReportItem> manageBindingForLDEPF(SCL scd, EPF epf) {
         return sclReportItems;
     }
 
+    @Override
+    public void epfPostProcessing(SCL scd) {
+        iedService.getFilteredIeds(scd, ied -> !ied.getName().contains("TEST"))
+                .forEach(tied -> ldeviceService.findLdevice(tied, tlDevice -> LDEVICE_LDEPF.equals(tlDevice.getInst()))
+                        .ifPresent(tlDevice -> tlDevice.getLN0().getDOI()
+                                .stream().filter(tdoi -> tdoi.getName().startsWith("InRef"))
```

Comment:
Reuse INREF prefix as constant already defined in LN0Adapter.
```suggestion
                                .stream().filter(tdoi -> tdoi.getName().startsWith(INREF_PREFIX))
```